test/Garnet.test/RespSetTest.cs

```diff
@@ -50,6 +50,27 @@ public void CanAddAndListMembers()
             Assert.AreEqual(expectedResponse, actualValue);
         }
 
+        [Test]
+        public void CanCheckIfMemberExistsInSet()
+        {
+            using var redis = ConnectionMultiplexer.Connect(TestUtils.GetConfig());
+            var db = redis.GetDatabase(0);
+            var key = new RedisKey("user1:set");
+
+            db.KeyDelete(key);
+
+            db.SetAdd(key, new RedisValue[] { "Hello", "World" });
+
+            var existingMemberExists = db.SetContains(key, "Hello");
+            Assert.IsTrue(existingMemberExists);
+
+            var nonExistingMemberExists = db.SetContains(key, "NonExistingMember");
+            Assert.IsFalse(nonExistingMemberExists);
+
+            var setDoesNotExist = db.SetContains("NonExistingSet", "AnyMember");
+            Assert.IsFalse(setDoesNotExist);
+        }
```

Comment:
Please add a test case or two for checking invalid # of arguments

Reply:
I added it in LightClient testcase, because if added here, the input requirements of db.SetContains are not met.